pyproject.toml

```diff
@@ -21,7 +21,7 @@ python = "^3.6"
 pyusb = "^1.0"
 pyserial = "^3.4"
 pyquaternion = "^0.9.5"
-zoloto = {version = ">=0.5,<0.6", optional = true}
+zoloto = {version = ">=0.5.2,<0.6", optional = true}
```

Comment:
Also no. We're fine to use 0.5.1 or 0.5.0 here, as long as we use 0.5.2 or greater in dev.